packages/@okta/vuepress-site/docs/guides/terraform-create-custom-auth-server/main/index.md

```diff
@@ -0,0 +1,407 @@
+
+For related conceptual information, see [Authorization server overview](/docs/concepts/auth-servers/).
+
+## Plan your implementation
+
+There are two types of Okta authorization servers:
+
+* **Org authorization server.** Every Okta org has a built-in authorization server called the org authorization server. The base URL for the org authorization server is `https://{<yourOktaDomain}`. This server provides basic functionality for Single Sign-On (SSO) using OpenID Connect apps or to get an access token that uses the Okta API scopes. You can't customize OAuth audience, claims, policies, or scopes, except [group claims](/docs/guides/customize-tokens-groups-claim/main/#add-a-groups-claim-for-the-org-authorization-server) and [user attributes](https://support.okta.com/help/s/article/How-to-add-custom-attributes-of-user-profile-as-claims-in-token?language=en_US).
+
+* **Custom authorization server.** Create your own server to support custom OAuth scopes, rules for granting scopes, and customizing the claims and policies.
+
+> **Note:** Okta creates a custom authorization server for every org named `default`. That default custom authorization server is **not** the org authorization server. Although it’s possible to manage the *default custom authorization server* in Terraform using the resource `okta_auth_server_default`, this article focuses on creating a new custom authorization server. A new authorization server addresses a broader range of use cases and encourages complete resource lifecycle management in Terraform.
+
+The first step is to understand the requirements for the custom server. These requirements include:
+
+* An identity token (authentication), resource access token (authorization), or both types of tokens.
+
+* Custom scopes for custom APIs or other private resources that are accessible by only some users.
+
+* Access to pre-defined Okta API scopes?
+
+The following table compares the capabilities of the org authorization server and custom authorization servers.
+
+| Capabilities | Org authorization server | Custom authorization server |
+| ------------ | ------------------------ | --------------------------- |
+| SSO with OpenID Connect | Yes | Yes |
+| Use Okta Developer SDKs & Widgets for SSO | Yes | Yes |
+| Retrieve the user profile in an identity token | Yes | Yes |
+| Apply authorization policies to custom APIs | – | Yes |
+| Add custom group or user attribute claims to tokens | – | Yes |
+| Add custom scopes or claims to tokens | – | Yes |
+| Integrate with an API Gateway | – | Yes |
+| Machine-to-machine or microservices | – | Yes |
+| Mint resource access tokens with the pre-defined Okta API scopes | Yes | – |
+
+
+Because the default org auth server is not configurable, the Okta Terraform provider supports only custom authorization servers.
```

Comment:
Made that change and reorganized the sentence so the subject is first.

> The Okta Terraform provider only supports custom authorization servers because the org authorization server is not configurable.

Change is in next commit.